tools/experiments/experiments.js

```diff
@@ -51,6 +55,7 @@ let ExperimentDef;
  */
 const CANARY_EXPERIMENT_ID = 'dev-channel';
 const RC_EXPERIMENT_ID = 'rc-channel';
+const RTV_EXPERIMENT_ID = 'rtv-channel';
```

Comment:
Based on the above comment, change `RTV_EXPERIMENT_ID` to `RTV_CHANNEL_ID`.

Reply:
Done